src/ApiClient.js

```diff
@@ -1,14 +1,46 @@
-import config from './config';
+var config = require('./config');
```

Comment:
Let's keep this consistent with the rest of the codebase (using ES6 import statements isn't a typo :P)